packages/hardhat-core/test/fixture-projects/compilation-import-non-existing-file/contracts/A.sol

```diff
@@ -0,0 +1,7 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.19;
+
+// For test purposes this file must not exist
+import "some-plugin/nonExistingFile.sol"
```

Comment:
Same comment about the dependency here. I'd rename it to `some-lib` to avoid confusions, and remove the `index.js` that adds a task.